test/hpr_packet_router_service_SUITE.erl

```diff
@@ -24,7 +25,8 @@
 %%--------------------------------------------------------------------
 all() ->
     [
-        session_test
+        session_test,
+        timeout_test
```

Comment:
There's one more test we need.
resetting the session.

after the regular `register -> session_offer -> session_init -> ok`
we have the `?SESSION_TIMER` of 30 minutes. 
that restarts `session_offer -> session_init -> ok`. 

That is when we're concerned about the response timeout.
We send the 2nd session_offer, and if they don't respond to it, nothing happens.

The way I understand things in this PR, after 30 mins we send the session_offer, and then check 5 minutes later if the session_key exists. But we don't check if it was the same session_key that already existed.